src/components/main.tsx

```diff
@@ -18,25 +22,60 @@ export function Main(): JSX.Element {
   }
 
   return (
-    <div className="text-center p-2 h-screen">
-      <div className="max-w-6xl text-left mx-auto ">
-        <div className="flex justify-between">
-          <DfxLogo />
-          <Connect />
+    <>
+      {/* MODALS */}
+      <StyledModal
+        type={StyledModalTypes.ALERT}
+        isVisible={needsSignUp}
+        onClose={() => {
+          // intentionally do nothing, as alert are closed via button
+        }}
```

Comment:
`onClose` should only be required on non-alert modals then.

Reply:
yes will remove it